examples/Data-class.R

```diff
@@ -1,4 +1,5 @@
 # create some data from the class 'Data'
-myData <- Data(x = c(0.1, 0.5, 1.5, 3, 6, 10, 10, 10),
-               y = c(0, 0, 0, 0, 0, 0, 1, 0),
-               doseGrid = c(0.1, 0.5, 1.5, 3, 6, seq(from = 10, to = 80, by = 2)))
\ No newline at end of file
+my_data <- Data(x = c(0.1, 0.5, 1.5, 3, 6, 10, 10, 10),
+                y = c(0, 0, 0, 0, 0, 0, 1, 0),
+                doseGrid = c(0.1, 0.5, 1.5, 3, 6,
+                             seq(from = 10, to = 80, by = 2)))
```

Comment:
```suggestion
my_data <- Data(
  x = c(0.1, 0.5, 1.5, 3, 6, 10, 10, 10),
  y = c(0, 0, 0, 0, 0, 0, 1, 0),
  doseGrid = c(
    0.1, 0.5, 1.5, 3, 6,
    seq(from = 10, to = 80, by = 2)
  )
)
```
-> code formatting as usual for long function calls please

Reply:
I did it intentionally as this is an example printed in the function man page, and in this case seems that non-code formatting is used for easy reading. See, e.g. `?read.table`. Would you like me to still change it?

Reply:
yeah I changed it now